POC-OOBE/OOBE-Sandbox/PowerToys Settings Sandbox/Services/ActivationService.cs

```diff
@@ -21,6 +25,88 @@ internal class ActivationService
 
         private object _lastActivationArgs;
 
+        private void newNotification()
+        {
+            
+            // In a real app, these would be initialized with actual data
```

Comment:
Remove template comment if not representative of below vars

Reply:
Done. Thank you!